packages/node_modules/@node-red/editor-client/src/js/ui/view.js

```diff
@@ -796,6 +798,7 @@ RED.view = (function() {
                         updateActiveNodes();
                         updateSelection();
                         redraw();
+                        if (auto === true) { point[0] = point[0] + ((nn.w < 100) ? 100 : nn.w) + gridSize * 2; }
```

Comment:
moves the location ready for the next node to be placed (based on width of node and grid spacing)